src/omdbapi.js

```diff
@@ -0,0 +1,20 @@
+const API_KEY = "6de06ef5"
```

Comment:
It's always better to put the API key outside of the code, e.g. in an environment variable, so that it's not included in the source published on GitHub.